classes/options-general.php

```diff
@@ -564,13 +564,13 @@ class='howto'><?php _e( 'Select "No Limit" to display all results within the sea
 
 													<tr valign="top">
 														<td><label
-																for="map_pages"><?php _e( 'Map Page IDs', 'simplemap' ); ?></label>
+																for="map_pages"><?php _e( 'Map Page IDs', 'SimpleMap' ); ?></label>
 														</td>
 														<td>
 															<input type="text" name="map_pages" id="map_pages" size="30"
 															       value="<?php echo esc_attr( $map_pages ); ?>"/><br/>
 															<small>
-																<em><?php _e( 'Enter the IDs of the pages/posts the map will appear on, separated by commas. The map scripts will only be loaded on those pages. Leave blank or enter 0 to load the scripts on all pages.', 'simplemap' ); ?></em>
+																<em><?php _e( 'Enter the IDs of the pages/posts the map will appear on, separated by commas. The map scripts will only be loaded on those pages. Leave blank or enter 0 to load the scripts on all pages.', 'SimpleMap' ); ?></em>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 290 characters

<!-- Comment by [SideCI](https://sideci.com) -->